types/xml-zero-lexer/xml-zero-lexer-tests.ts

```diff
@@ -0,0 +1,28 @@
+import Lexx, { NodeTypes, NodeTypeKeys } from 'xml-zero-lexer';
+
+const xml = '<p>his divine shadow</p>';
+
+const options: Lexx.Options = { jsx: true, html: true, blackholes: ["script"] };
+
+Lexx(xml, options);
+
+const tokens = Lexx(xml);
+
+// 'tokens' is now:
+//   [
+//     [NodeTypes.ELEMENT_NODE, 1, 2],
+//     [NodeTypes.TEXT_NODE, 3, 20],
+//     [NodeTypes.CLOSE_ELEMENT],
+//   ]
+
+xml.substring(tokens[0][1], tokens[0][2]);
+// would return "p"
+
+xml.substring(tokens[1][1], tokens[1][2]);
+// would return "his divine shadow"
+
+for (let index = 0; index < NodeTypeKeys.length; index++) {
+	const NodeTypeKey = NodeTypeKeys[index];
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/xml-zero-lexer/xml-zero-lexer-tests.ts;
  Line: 25;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.